src/functions/negate.js

```diff
@@ -3,13 +3,39 @@ import {wrap} from "../core/wrap";
 // Get the complement of a function.
 export const negate = wrap({
     name: "negate",
+    summary: "Get the logical negation of a predicate function.",
```

Comment:
The summary is meant to be a short string that I'm leaning toward including even in prod builds. @jpsullivan thoughts?